staging/src/k8s.io/client-go/tools/record/events_cache.go

```diff
@@ -169,6 +169,7 @@ func EventAggregatorByReasonFunc(event *v1.Event) (string, string) {
 		event.InvolvedObject.Kind,
 		event.InvolvedObject.Namespace,
 		event.InvolvedObject.Name,
+		event.InvolvedObject.FieldPath,
```

Comment:
what is the expected change in event count here? do we *want* to create the same event / message for multiple containers in a pod? that will  use up more of the per-pod event quota and mean that events with completely different messages / reasons can get discarded.

Reply:
cc @derekwaynecarr @bobbypage  in case you have a person who is more familiar with current state of node event reporting who would know how this change is likely to impact kubelet

Reply:
> do we _want_ to create the same event / message for multiple containers in a pod?

from the current kubelet logic, it will create event for each container in same pod with different `FieldPath` property. 
The comment also said event that varies only by message is treated as same event and can be aggregated. 
https://github.com/kubernetes/kubernetes/blob/d3479eeeac1727886bc9496d9d119b9adff52c32/staging/src/k8s.io/client-go/tools/record/events_cache.go#L39-L42

Reply:
@pacoxu Could you please help take a look at this when you have a moment? thx